src/components/BankTransactionsLoader/BankTransactionsLoader.tsx

```diff
@@ -8,7 +8,7 @@ import { SkeletonTableLoader } from '../SkeletonTableLoader'
 
 export const BankTransactionsLoader = ({
   isLoading = true,
-  showTooltips
+  showTooltips,
```

Comment:
This would certainly be in a different PR, but I am curious what you think about potentially using [Jotai](https://jotai.org/) at some point, since I did a lot of prop drilling for the tooltips / receipt upload props.

Reply:
Definitely there are good use cases for Jotai. The only two minor concerns I have are:
- NPM package weights a bit, ~427kB... but still worth to use imho
- we will need to be careful with mixing atoms with specific local states and data later sent to API

Reply:
We had a brief discussion and agreed we should just move some things into context at this stage instead, but will revisit Jotai if we later have issues with too many re-renders (which we have not thus far).